modules/rosa-sts-aws-requirements.adoc

```diff
@@ -21,6 +21,100 @@ The following prerequisites must be complete before you deploy a {product-title}
 Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
 ====
 
+//
+// Start work for OSDOCS-3198
+//
+[id="rosa-associating-account_{context}"]
+== Associating your AWS account
+
+Performing {product-title} (ROSA) cluster provisioning tasks in {cluster-manager} require linked `ocm-role` and `user-role` resources to your AWS account by using your Amazon Resource Name (ARN).
+
+.Prerequisites
+
+* You have an AWS account.
+* You are using {cluster-manager-url} to create clusters.
+* You have the permissions required to install AWS account-wide roles.
+* You have installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your installation host.
+* You have created your `ocm-role` and `user-role` IAM roles.
+
+.Procedures
+
+1. From your desired CLI, link your `ocm-role` resource to your AWS account by using your Amazon Resource Name (ARN):
++
+[NOTE]
+====
+You must have organization administrator privileges to run the `rosa link` command. After you link the `ocm-role` resource with your AWS account, it is visible for all users in the organization.
+====
++
+[source,terminal]
+----
+rosa link ocm-role --role-arn <arn>
+----
++
+.Example output
+[source,terminal]
+----
+I: Linking OCM role
+? Link the '<AWS ACCOUNT ID>` role with organization '<ORG ID>'? Yes
+I: Successfully linked role-arn '<AWS ACCOUNT ID>' with organization account '<ORG ID>'
+----
+1. From your desired CLI, link your `user-role` resource to your AWS account by using your Amazon Resource Name (ARN):
++
+[source,terminal]
+----
+rosa link user-role --role-arn <arn>
+----
++
+.Example output
+[source,terminal]
+----
+I: Linking User role
+? Link the 'arn:aws:iam::<ARN>:role/ManagedOpenShift-User-Role-125' role with organization '<AWS ID>'? Yes
+I: Successfully linked role-arn 'arn:aws:iam::<ARN>:role/ManagedOpenShift-User-Role-125' with organization account '<AWS ID>'
+----
+
+[id="rosa-associating-multiple-account_{context}"]
+=== Associating multiple AWS accounts with your {cluster-manager} organization
+
+You can associate multiple AWS accounts with your {cluster-manager-first} organization. Associating multiple accounts enables you to create {product-title} (ROSA) clusters on any of the associated AWS accounts from your {cluster-manager} organization.
```

Comment:
@EricPonvelle following a recent discussion about the terminology, please change all references to "OpenShift Cluster Manager organization" in this PR to "Red Hat organization". I won't highlight them all in individual comments. Thanks!